Source/JavaScriptCore/runtime/ExceptionHelpers.cpp

```diff
@@ -340,6 +340,11 @@ JSObject* createErrorForInvalidGlobalVarDeclaration(JSGlobalObject* globalObject
     return createTypeError(globalObject, makeString("Can't declare global variable '"_s, ident.string(), "': global object must be extensible"_s));
 }
 
+JSObject* createTDZError(JSGlobalObject* globalObject, const Identifier& ident)
```

Comment:
Doesn't seem like this needs to be an `Identifier` it probably already is in most cases but we should make this just a `StringView`. It's pretty expensive to make a new AtomString so we should avoid it if it's unnecessary.